bundle/compliance/cis_azure/rules/cis_7_2/test.rego

```diff
@@ -0,0 +1,121 @@
+package compliance.cis_azure.rules.cis_7_2
+
+import data.compliance.policy.azure.data_adapter
+import data.lib.test
+
+valid_managed_disk = {
```

Comment:
should be under test_data

Reply:
Should I really move it if it's only used in this file?

Reply:
Yes this is mostly for consistency and readability, if we'll get reusability of it in the future for free is just a plus